lib/sinon/spy.js

```diff
@@ -308,6 +314,16 @@ var spyApi = {
         return fake;
     },
 
+    matchingFakes: function (args, strict) {
+        if (!this.fakes) {
+            return undefined;
```

Comment:
This could return an empty array, which would make it easier to write elegant consumers of the returned value